docs/core/tools/dotnet-restore.md

```diff
@@ -199,14 +199,16 @@ Starting in .NET 8, `dotnet restore` includes NuGet security auditing. This audi
 
 To opt out of the security auditing, set the `<NuGetAudit>` MSBuild property to `false` in your project file.
 
-To retrieve the known vulnerability dataset, ensure that you have the NuGet.org central registry defined as one of your package sources:
+To retrieve the known vulnerability dataset from the NuGet.org central registry, define the following in the *nuget.config* file:
```

Comment:
When NuGetAudit was added in .NET 8, using a NuGet feed with vulnerability data (like nuget.org) as a package source was the only way to get it to work. But in .NET 9 we added `auditSources`, so customers can avoid nuget.org as a package source for supply chain security reasons, but still use audit. NuGet's docs on auditing packages explains the differences between audit sources and package sources, along with other configuration options.

IMO this makes it sound like using nuget.org as a package source is the only way to use NuGetAudit, which I'm concerned will discourage people who don't use nuget.org for security reasons.